testomania/src/main/kotlin/com/earth/testomania/ui/theme/Color.kt

```diff
@@ -18,4 +18,6 @@ val DialogBkgDark = Color(0xFF222222)
 val DialogBkgLight = Color(0xFFEEEEEE)
 
 val LightRed = Color(0xffffb4ab)
-val DarkRed = Color(0xffba1a1a)
\ No newline at end of file
+val DarkRed = Color(0xffba1a1a)
+val BottomBarColorDark = Color(0xff262c39)
+val BottomBarColorLight = Color(0xffededee)
```

Comment:
btw, we should try to use colors only from Kiwi. this might have bad contrast on dark theme

Reply:
hmm will try to use kiwi's colors